src/Arcus.Security.Core/Arcus.Security.Core.csproj

```diff
@@ -18,24 +18,32 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Allows (for now) preview packages, as we are referencing Arcus' preview .NET 8 package. -->
+    <NoWarn>NU5104</NoWarn>
+  </PropertyGroup>
+
   <ItemGroup>
     <None Include="..\..\README.md" Pack="true" PackagePath="\" />
     <None Include="..\..\LICENSE" Pack="true" PackagePath="\" />
     <None Include="..\..\docs\static\img\icon.png" Pack="true" PackagePath="\"/>
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
-    <PackageReference Include="Arcus.Observability.Telemetry.Core" Version="[2.4.0,3.0.0)" />
-    <PackageReference Include="Guard.Net" Version="2.0.0" />
+  <ItemGroup >
+    <PackageReference Include="Arcus.Observability.Telemetry.Core" Version="[3.0.0-preview-1-net8.0,4.0.0)" />
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
-    <PackageReference Include="Arcus.Observability.Telemetry.Core" Version="[2.0.0,3.0.0)" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net8.0'">
+    <PackageReference Include="Guard.Net" Version="2.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0' and '$(TargetFramework)' != 'net8.0'">
+    <!--<PackageReference Include="Arcus.Observability.Telemetry.Core" Version="[2.0.0,3.0.0)" />-->
     <PackageReference Include="Guard.Net" Version="1.2.0" />
```

Comment:
Do we need Guard.NET 1.2.0 when building for .NET Standard ?

Reply:
Yep, that is correct.

Reply:
But updated the package to v3, so it is compatible to both .NET standard and .NET 8. 👍

Reply:
Updated to Guard v3 here: https://github.com/arcus-azure/arcus.security/pull/421/commits/d24b66901b7efe25e896f007b5b6a323722a58c4